### PR TITLE
Assignment6.2

### PR DESCRIPTION
**Pull request by SophieLAPIDOC**

### CODE REVIEW

**Comment by rbwatson** [comment]
Rebased the PR/feature branch with the current version of the `main` branch.

File: docs/first-use_SophieL.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+So, here we are. It's Monday morning. You've barely had time to take a sip of your first coffee and already life is out of control.
+
+So many sticky notes, so little time, as they say.
+
+Time to take control of the chaos. Time to create your first task and set a rminder with To-Do service.
+
+It will be ready by the time you next cup of coffee is ready to enjoy.
+
+## Prerequisite
+To use the To-Do service, you need a user account. Read all about it in our first tutorial [Enrol a new user](https://uwc2-apidoc.github.io/to-do-service-au24/tutorials/enroll-a-new-user.html).
```
Comment (rbwatson) [inline docs/first-use_SophieL.md:11]:
```suggestion
To use the To-Do service, you need a user account. Read all about it in our first tutorial [Enroll a new user](https://uwc2-apidoc.github.io/to-do-service-au24/tutorials/enroll-a-new-user.html).

```

File: docs/first-use_SophieL.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+   For this example, enter -3.
+9. Select **Save** to save your first task.
+
+Time to boil the kettle and freshen up that cuppa. In a few minutes, To-Do will send you an email notification to remind you of Amy's appointment.
+
+This is just the beginning. You can repeat these simple steps to create more tasks and reminder. 
+Make sure you re-read our tutorial [Enrol a new user](https://uwc2-apidoc.github.io/to-do-service-au24/tutorials/enroll-a-new-user.html). How about creating more user accounts for the family and assigning chores?
+
+YOu can raise your cup to that.
```
Comment (rbwatson) [inline docs/first-use_SophieL.md:33]:
```suggestion
You can raise your cup to that.

```

File: docs/index.md
Comment (rbwatson) [inline docs/index.md:1]: I added your topic to the overview/landing page to make it accessible.

**Review by rbwatson — CHANGES_REQUESTED** [review]
I like the idea of this topic, but it's not a quick start topic.

This topic is written as more of a visualization of what your (the customer's) app could look like after you've integrated the to-do-service into it.

So, instead of calling it a quickstart, call it a _use-case example_ of how the service would look when integrated into an app. Then conclude by linking them to a quickstart or one of the **Add new user** or **Add new task** topics to show how they can start using it

**Comment by rbwatson** [comment]
A great topic!

Review the comments in the review and update your topic. Build and test your updated topics, squash the commits so there's only one and re-request a review.

**Comment by SophieLAPIDOC** [comment]
Hi Rob, please review. I have accepted your suggestions, then squashed commits via GitHub desktop. I will wait for your feedback before attempting to modify assignment 6.1. Hopefully, I have not destroyed everybody's work.

**Comment by SophieLAPIDOC** [comment]
Hi Rob, please review. I have accepted your suggestions, then squashed commits via GitHub desktop. I will wait for your feedback before attempting to modify assignment 6.1. Hopefully, I have not destroyed everybody's work.

**Pull request closed without merging**